src/components/Graph.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Plot from "./Plot";
+import PropTypes from "prop-types";
+import { Container, Alert, AlertTitle, Grid, Paper } from "@mui/material";
+import Deposits from "../components/Deposits";
+
+function Graph({
+  currentTab,
+  graphs,
+  sortIntoGrid,
+  configIndicated,
+  configIsLoading,
+}) {
+  if (!configIndicated) {
+    return (
+      <Alert severity="warning">
+        <AlertTitle fontSize={25}>Uwaga</AlertTitle>
+        Aby wyświetlić dane załaduj config korzystając z menu po lewej stronie
+      </Alert>
+    );
+  }
+  if (configIsLoading) {
+    return (
+      <Alert severity="info">
+        <AlertTitle fontSize={25}>Informacja</AlertTitle>
+        Plik konfiguracyjny się ładuje
+      </Alert>
+    );
+  }
+  return (
+    <Container maxWidth="lg" sx={{ mt: 4, mb: 4 }}>
+      {graphs.map((graph) => (
+        <Grid container spacing={3} sx={{ m: 1 }}>
+          <Grid item xs={12} md={8} lg={9}>
+            <Paper
+              sx={{
+                p: 3,
+                display: "flex",
+                flexDirection: "column",
+              }}
+            >
+              <Plot key={graph.title} chart={graph} aspect={2.5} />
+            </Paper>
+          </Grid>
+          <Grid item xs={12} md={4} lg={3}>
+            <Paper
+              sx={{
+                p: 3,
+                display: "flex",
+                flexDirection: "column",
+              }}
+            >
+              <Deposits />
```

Comment:
Kumpel podziel ten depozyt na 2 kolumny i zrob tak zeby wyswietlao tablice klucz wartosc ktora przekazesz w argumencie ktora przekazesz w contenerze do Graph i backend se juz podepnie pod to